parquet_derive/src/lib.rs

```diff
@@ -160,6 +167,7 @@ pub fn parquet_record_writer(input: proc_macro::TokenStream) -> proc_macro::Toke
 ///
 /// pub fn read_some_records() -> Vec<ACompleteRecord> {
 ///   let mut samples: Vec<ACompleteRecord> = Vec::new();
+///   let mut file = tempfile().unwrap();
```

Comment:
```suggestion
///   let mut file = todo!();
```
And same above, would avoid a dependency

Reply:
It does not work.
```rust
error[E0277]: the trait bound `(): std::io::Write` is not satisfied
   --> parquet_derive/src/lib.rs:73:19
    |
34  |  let mut writer = SerializedFileWriter::new(file, schema, Default::default()).unwrap();
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::io::Write` is not implemented for `()`
    |
note: required by a bound in `SerializedFileWriter`
   --> /Users/hwj/Desktop/rust/arrow-rs/parquet/src/file/writer.rs:141:36
    |
141 | pub struct SerializedFileWriter<W: Write> {
    |                                    ^^^^^ required by this bound in `SerializedFileWriter`
```

Reply:
I use `std::fs::File` to avoid a dependency